intel_owl/settings.py

```diff
@@ -50,6 +50,7 @@
     "durin",
     "guardian",
     "api_app.apps.ApiAppConfig",
+    "api_app.analyzers_manager.apps.AnalyzersManagerConfig",
```

Comment:
Since the `api_app` is a django app and the `analyzers_manager` only a sub-app, we shouldn't add it here.

This also means no need for the `analyzers_manager/apps.py` file.

Same is true for `connectors_manager`, so ping @sp35.

Reply:
How would django recognize this sub app then (e.g. for making migrations)?

Reply:
Even I made this change after finding that a sub-app is independent from the main app and would require it to be added in the settings.py separately. I'll test it out without this line as well once

Reply:
AFAIK and I have used it like this before, just adding the "api_app" to the list should be sufficient. Django recursively traverses all sub-directories.

Reply:
@m0mosenpai when you ran `makemigrations`, did it create migration files inside `api_app/migrations` or `api_app/analyzers_manager/migrations/` ?

Reply:
Yeah, i got a `0001_initial.py` in `analyzers_manager/migrations` and a `0005_auto_xxxx.py` in `api_app/migrations`

Reply:
okay then that's good so we should keep the line including the sub-app in `INSTALLED_APPS`.